main/tests/server/src/com/google/refine/importers/XmlImportUtilitiesTests.java

```diff
@@ -623,41 +551,27 @@ public void processRecordsFromComplexXmlStructure() throws IOException {
     }
 
     // ----------------helpers-------------
-    public void loadSampleXml() {
+    public void loadSampleXml() throws Exception {
```

Comment:
```suggestion
    public void loadSampleXml() {
```

Doesn't throw with the charset fix